simulator-core/build.gradle

```diff
@@ -48,4 +59,60 @@ dependencies {
 
 test {
     useJUnitPlatform()
+}
+
+jar {
+    manifest {
+        attributes(
+                'Main-Class': 'com.evbox.everon.ocpp.simulator.SimulatorLauncher'
+        )
+    }
+
+    from {
+        configurations.compile.collect { it.isDirectory() ? it : zipTree(it) }
+    }
+}
+
+task sourcesJar(type: Jar) {
+    from sourceSets.main.allJava
+    archiveClassifier = 'sources'
+}
+
+task javadocJar(type: Jar) {
+    from javadoc
+    archiveClassifier = 'javadoc'
+}
+
+publishing {
+    publications {
+        StationSimulator(MavenPublication) {
+            from components.java
+            artifact sourcesJar
+            artifact javadocJar
+            groupId 'io.github.evbox'
+            artifactId 'ocpp-station-simulator'
+            version '0.1'
+        }
+    }
+}
+
+bintray {
+    user = project.hasProperty('user') ? project.property('user') : null
```

Comment:
won't project.property('user') get evaluated to null in case of the missed property?

Reply:
Yes that's the point, if you try to run ./gradlew bintrayUpload without setting these properties you will get

1: Task failed with an exception.
-----------
* What went wrong:
Execution failed for task ':simulator-core:bintrayUpload'.
> Could not create package 'null/maven/station-simulator': HTTP/1.1 401 Unauthorized [message:This resource requires authentication]

Reply:
I mean user = **project.hasProperty('user') ? project.property('user') : null**
should be the same as **user = project.property('user')**

Reply:
But then If I just simply do ./gradlew build without invoking this task I will get

```
* What went wrong:
A problem occurred evaluating project ':simulator-core'.
> Could not get unknown property 'user' for project ':simulator-core' of type org.gradle.api.Project.
```

Reply:
Ok